frontend/user/profile_mobile.html

```diff
@@ -0,0 +1,37 @@
+<md-dialog id="user-profile-size">
+    <div class="user-profile-dialog">
+        <div class="user-profile-cover">
+            <div md-colors="{background: 'default-teal-600'}">
+                <img>
+            </div>
+            <div id="user-profile-img">
+                <img ng-src="{{profileCtrl.user.photo_url}}" />
+            </div>
+        </div>
+        <div class="user-profile-content">
```

Comment:
two divs. one inside the other. with no sibling content. couldnt just be one div?